### ISSUE TITLE
Bisha Airport OEBH

### ISSUE THREAD

**Issue by M3RKZ** [InfiniteFlightAirportEditing/Airports#3332]
# Airport Name

< Bisha >

# Country?

< Saudia Arabia  >

# Improvements that need to be made?

< Adding Taxiways and Parking and more fix>

# Are you working on this airport?

< yes >

# Airport Priority? (A380, 10000ft+ Runway)

< Enter text here >

**Comment by M3RKZ** [comment]
Hello is that mean another editor is already working on the airport ?

**Comment by brunocr98** [comment]
No, you're assigned to this issue because you 1. You created it and 2. You said "yes" in the 3rd question.

**Comment by M3RKZ** [comment]
yes im working... ok thanks

**Comment by kilt46** [comment]
![image](https://user-images.githubusercontent.com/14023167/47619954-3ac7b300-dab2-11e8-8e4d-0e31d7aea6ff.png)
